pkg/docker/client.go

```diff
@@ -150,7 +150,7 @@ func (dm *DockerManager) StartContainer(options docker_client.CreateContainerOpt
 	}
 
 	// Run the container
-	err = dm.Client.StartContainer(container.ID, options.HostConfig)
+	err = dm.Client.StartContainer(container.ID, nil)
```

Comment:
Why need this change? It will affect [here](https://github.com/caicloud/cyclone/blob/master/pkg/worker/stage/stage.go#L141-L158)

Reply:
can we update the docker in worker to `docker:1.13.1-dind`.
[setting host configuration options when the container starts is deprecated](https://docs.docker.com/engine/deprecated/#hostconfig-at-api-container-start)